src/main/java/Steps/MyStepdefs.java

```diff
@@ -24,7 +24,9 @@
 public class MyStepdefs {
     WebDriver driver;
     UpdateForm updateForm;
-    Post myUser;
+    User myUser;
+    Gson gson = new Gson();;
```

Comment:
As we use this objects in multiply methods - it's better to move them as class variables